### DISCUSSION TITLE
pip install error

### DISCUSSION THREAD

**Discussion by joaosppy** [seleniumbase/SeleniumBase#1763] [Q&A] [answered]
I tried to install by pip but I couldn't.

Google Colab
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
ipython 7.9.0 requires jedi>=0.10, which is not installed.
cvxpy 1.2.3 requires setuptools<=64.0.2, but you have setuptools 67.3.2 which is incompatible.

Windows 10 (Anaconda)
ERROR: To modify pip, please run the following command:
C:\Users\...\anaconda3\python.exe -m pip install seleniumbase --force-reinstall

Ubuntu 22 (Anaconda)
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
spyder 5.3.3 requires pyqt5<5.16, which is not installed.
spyder 5.3.3 requires pyqtwebengine<5.16, which is not installed.
conda-repo-cli 1.0.20 requires clyent==1.2.1, but you have clyent 1.2.2 which is incompatible.
conda-repo-cli 1.0.20 requires nbformat==5.4.0, but you have nbformat 5.5.0 which is incompatible.
conda-repo-cli 1.0.20 requires requests==2.28.1, but you have requests 2.28.2 which is incompatible.

**Comment by mdmintz** [comment, marked as answer]
It looks like you're trying to install different packages with conflicting dependencies. You may want to use a virtual environment to isolate packages and dependencies: https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/virtualenv_instructions.md
